frontend/public/components/masthead-toolbar.jsx

```diff
@@ -304,6 +306,15 @@ class MastheadToolbar_ extends React.Component {
               },
             ]
           : []),
+        ...(reportBugLink
+          ? [
+              {
+                label: 'Report Bug',
```

Comment:
Can we toggle this text so that when we have a supported version it shows open a case? Not open a bug.

Reply:
@sferich888 I pushed an update to change the label based on what link we're showing:

<img width="508" alt="Dashboards · OKD 2019-10-23 19-38-23" src="https://user-images.githubusercontent.com/1167259/67441911-c93f1980-f5cc-11e9-808d-37eac22a5733.png">

Reply:
Thanks @spadgett